### ISSUE TITLE
Aggiunta altre distribuzione dei pesi e dei bias per tensorboard 

### ISSUE THREAD

**Issue by MickPerl** [jjocram/expanded-mnist#4]
for name, param in model.named_parameters(): 
			&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; name = name.replace('.', '/') 
			&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; tb.add_histogram('every epoch_' + name, param.data.cpu().detach().numpy(), epoch) 
			&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; tb.add_histogram('every epoch_' + name + '/grad', param.grad.data.cpu().numpy(), epoch)

subito dopo:
`tb.add_scalar("Loss every epoch", mean_loss_current_epoch, epoch)`
